src/msg/async/Stack.cc

```diff
@@ -121,17 +118,18 @@ void NetworkStack::start()
     return ;
   }
 
-  for (unsigned i = 0; i < num_workers; ++i) {
-    if (workers[i]->is_init())
+  for (Worker* worker : workers) {
+    if (worker->is_init())
       continue;
-    std::function<void ()> thread = add_thread(i);
-    spawn_worker(i, std::move(thread));
+    std::function<void ()> thread = add_thread(worker);
```

Comment:
`thread` is a thread function, its naming is a little werid..

Reply:
@runsisi eliminated the local variable. could you take another look?